HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1694,21 +1694,23 @@ def self.apply_detailed_performance_data_for_var_speed_systems(hpxml_bldg)
     end
   end
 
-  def self.apply_hvac_control(hpxml_bldg, schedules_file)
+  def self.apply_hvac_control(hpxml_bldg, schedules_file, eri_version)
     hpxml_bldg.hvac_controls.each do |hvac_control|
       schedules_file_includes_heating_setpoint_temp = (schedules_file.nil? ? false : schedules_file.includes_col_name(SchedulesFile::ColumnHeatingSetpoint))
       if hvac_control.heating_setpoint_temp.nil? && hvac_control.weekday_heating_setpoints.nil? && !schedules_file_includes_heating_setpoint_temp
         # No heating setpoints; set a default heating setpoint for, e.g., natural ventilation
-        htg_sp, _htg_setback_sp, _htg_setback_hrs_per_week, _htg_setback_start_hr = HVAC.get_default_heating_setpoint(HPXML::HVACControlTypeManual)
-        hvac_control.heating_setpoint_temp = htg_sp
+        htg_weekday_setpoints, htg_weekend_setpoints = HVAC.get_default_heating_setpoint(HPXML::HVACControlTypeManual, eri_version)
+        hvac_control.weekday_heating_setpoints = htg_weekday_setpoints
+        hvac_control.weekend_heating_setpoints = htg_weekend_setpoints
```

Comment:
@bpark1327 Since OS-HPXML only ever uses the "manual" thermostat setpoint defaults, you can simplify the code by assuming there are constant setpoints. For example (code is untested):

```suggestion
        htg_weekday_setpoints, htg_weekend_setpoints = HVAC.get_default_heating_setpoint(HPXML::HVACControlTypeManual, eri_version)
        if htg_weekday_setpoints.split(',').uniq.size == 1 && htg_weekend_setpoints.split(',').uniq.size == 1
          hvac_control.weekday_heating_setpoints = htg_weekday_setpoints
          hvac_control.weekend_heating_setpoints = htg_weekend_setpoints
        else
          fail 'Unexpected setpoints.'
        end
```

You could make similar changes below and in airflow.rb.

Reply:
Sorry, I didn't have it right the first time. I _think_ I have it right now (but it's still untested).